src/redux/store.js

```diff
@@ -0,0 +1,9 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    books: bookReducer,
+  },
+});
+export default store;
```

Comment:
Nice job overall 💯  but you should import `categories` and use `configureStore` to create a reducer. Also  `books` and `categories` should be added to the `configureStore` method. For now you only have `books` 🔍

Reply:
Clearly this was not necessary. Anyways thanks for the review

Reply:
Not according to the project requirements @veronica365 👇🏽 

![Screenshot 2023-04-05 at 07 13 51](https://user-images.githubusercontent.com/45739963/230064514-1634c14d-618d-410c-b946-bb37fef4ecfa.png)

So I'm just doing my job, please be respectful. 😢